include/internal/param_build_set.h

```diff
@@ -12,6 +12,8 @@
 
 int ossl_param_build_set_int(OSSL_PARAM_BLD *bld, OSSL_PARAM *p,
                              const char *key, int num);
+int ossl_param_build_set_long(OSSL_PARAM_BLD *bld, OSSL_PARAM *p,
+                              const char *key, int long);
```

Comment:
Is `int long` even legal?  `long` is a modified, `int` is the type.
What about `long int num` or `long num`?

Reply:
In a declaration, this is perfectly legal, you're allowed to skip the variable name :wink:
In the implementation, I have `long num`, so I'll update this declaration accordingly.

Reply:
It wasn't the lack of name I wondered about `int` before `long` is a little abnormal.
It is legal, both are type specifiers and the standard lets them be in any order.